api/fasit.go

```diff
@@ -252,27 +257,31 @@ func getResourceIds(usedResources []NaisResource) (usedResourceIds []int) {
 	return usedResourceIds
 }
 
-func fetchFasitResources(fasit FasitClientAdapter, deploymentRequest NaisDeploymentRequest, manifest NaisManifest) (naisresources []NaisResource, err error) {
-	resourceRequests := DefaultResourceRequests()
-	for _, resource := range manifest.FasitResources.Used {
-		resourceRequests = append(resourceRequests, ResourceRequest{
+func FetchFasitResources(fasit FasitClientAdapter, application string, environment string, zone string, usedResources []UsedResource) (naisresources []NaisResource, err error) {
+	defaultResources := DefaultResourceRequests()
+
+	defaultResourcesCount := len(defaultResources)
+	resourceRequests := append(DefaultResourceRequests(), make([]ResourceRequest, len(usedResources))...)
+
+	for index, resource := range usedResources {
+		resourceRequests[index+defaultResourcesCount] = ResourceRequest{
```

Comment:
Why not use `append()`? Is it because you're defining an array and not a slice?

Reply:
I think it could be simplified with something like this (pseudo):
```
func FetchFasitResources(fasit FasitClientAdapter, application string, environment string, zone string, usedResources []UsedResource) (naisresources []NaisResource, err error) {
	resourceRequests := DefaultResourceRequests()

	for _, resource := range usedResources {
		resourceRequests = append(resourceRequests, ResourceRequest{
			Alias:        resource.Alias,
			ResourceType: resource.ResourceType,
			PropertyMap:  resource.PropertyMap,
		})
	}
...
```

Reply:
The reasoning behind this was to avoid unnecessary reallocations of the array, I can change it to use append if that's preferred

Reply:
yeah, although I agree on the reallocations part (I have a C mindset...), golang itself recommends the usage of slices over arrays, so I think `append()` will do just fine